src/hierarchies/lin_reg_uni_hierarchy.h

```diff
@@ -12,89 +12,61 @@
 #include "hierarchy_prior.pb.h"
 #include "marginal_state.pb.h"
 
-class LinRegUniHierarchy : public DependentHierarchy {
- public:
-  struct State {
-    Eigen::VectorXd regression_coeffs;
-    double var;
-  };
-  struct Hyperparams {
-    Eigen::VectorXd mean;
-    Eigen::MatrixXd var_scaling;
-    Eigen::MatrixXd var_scaling_inv;
-    double shape;
-    double scale;
-  };
+namespace LinReg {
```

Comment:
This should be LinRegUni for consistency. Or otherwise we change the name of the class to this